dynamic_rest/filters.py

```diff
@@ -742,9 +820,27 @@ def ordering_for(self, term, view):
         for segment in serializer_chain[:-1]:
             field = serializer.get_all_fields().get(segment)
 
+            # If its a JSONField, construct a RawSQL command in the form
+            # of 'jsonField->{}'.format('nestedField')' or
+            # 'jsonField->>{}->{}'.format('nested','doubleNested')
+            if field and isinstance(field, JSONField):
+                json_chain_start = str(segment)
+                json_chain = ''
+                nested = []
+                first = True
+                for nterm in serializer_chain[1:]:
+                    if first:
+                        json_chain += '->>%s'
+                        first = False
+                    else:
+                        json_chain = '->%s'+json_chain
```

Comment:
```suggestion
                        json_chain = '->%s' + json_chain
```

Curious, did this pass linting?

Reply:
Huh, yeah it did. I thought the linting rules were pretty strict so I'm also surprised.

Reply:
Fixed!